automagica/activities.py

```diff
@@ -9818,9 +9833,14 @@ def detect_vision(element_id, detect_target=True):
     config_path = os.path.join(os.path.expanduser("~"), "automagica.json")
 
     # Read JSON
-    with open(config_path) as json_file:
-        local_data = json.load(json_file)
-        api_key = str(local_data["bot_secret"])
```

Comment:
Its not working for me everytime when i run i am getting Bot secret key error. can you please help it out